SKILL.tsv

```diff
@@ -4991,7 +4991,7 @@ SKILL_20150729_004990	$Increases the range of [Melstis] to 80.
 SKILL_20150729_004991	$Increases the maximum number of zombies from [Zombify] by 1.
 SKILL_20150730_004992	Summon the devil of card inserted in Grimore. Character's INT, SPR affects the ATK and DEF of summoned devil. Ability increases by 10% per card star. Grimore can only equip Demon type monster card.
 SKILL_20150730_004993	Make Shogos molded based on the card inserted in Necronomicon. Character's INT, SPR affects the ATK and DEF of summoned devil. Ability increases by 10% per card star. Necronomicon can equip only Animal, Plant, and Mutant type cards.
-SKILL_20150730_004994	Companion bites and holds on to target. Target's Physical DEF decreases and bonus chance of critical is applied.
+SKILL_20150730_004994	Your companion bites and holds onto a target. Target's Physical DEF decreases and a bonus chance of critical is applied.
```

Comment:
Target's Physical DEF decreases and a bonus chance of critical is applied. -> Decreases the physical defense of the bitten target. A bonus chance of critical/critical chance bonus applies when attacking the target.